cmd/set/account_password/account_password.go

```diff
@@ -0,0 +1,91 @@
+package account_password
```

Comment:
Change the package name and file name to `accountPassword` instead of `account_password`